dev-haskell/dump-core/dump-core-0.1.3.2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
`polysemy` has (or at least used to have) optional `dump-core` dependency. We recently removed `dump-core` package due to problems with current ghc version. Was it fixed since?

`use.mask` masked broken USE flag.

Reply:
I justisswd that dump-core used to exist and was taken out.

I'll remove my dump-core stuff